tests/test_object_states.py

```diff
@@ -758,7 +758,7 @@ def test_particle_source():
 
     # Cannot set this state
     with pytest.raises(NotImplementedError):
-        sink.states[ParticleSource].set_value(water_system, True)
+        sink.states[ParticleSource].set_value(True)
```

Comment:
oh this is an InstrinsicObjectState so we expect to raise a NotImplementedError, correct?